Source/Editor/DualityEditor/Controls/AutoExpandTreeView.cs

```diff
@@ -0,0 +1,68 @@
+using Aga.Controls.Tree;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Duality.Editor.Controls
+{
+	public class AutoExpandTreeView : TreeViewAdv
+	{
+		public int DragOverExpandDelay
```

Comment:
Looks good semantically! Style-wise, for property implementations that short, the single-line form is preferred:

```CSharp
private Timer autoExpandDelayTimer = new Timer();

public int AutoExpandDelay
{
    get { return this.autoExpandDelayTimer.Interval; }
    set { this.autoExpandDelayTimer.Interval = value; }
}
```

In other cases, we just omit the newline between getter and setter.